src/lang/english.txt

```diff
@@ -4142,6 +4142,11 @@ STR_VEHICLE_STATUS_HEADING_FOR_WAYPOINT_VEL                     :{LTBLUE}Heading
 STR_VEHICLE_STATUS_HEADING_FOR_DEPOT_VEL                        :{ORANGE}Heading for {DEPOT}, {VELOCITY}
 STR_VEHICLE_STATUS_HEADING_FOR_DEPOT_SERVICE_VEL                :{LTBLUE}Service at {DEPOT}, {VELOCITY}
 
+STR_VEHICLE_STATUS_CANT_REACH_STATION_VEL                       :{LTBLUE}Can't reach {STATION}, {VELOCITY}
```

Comment:
There aren't a lot of contractions in the strings at the moment, I wonder whether `Cannot reach...` would be better